cmd/commands/add.go

```diff
@@ -64,6 +64,7 @@ Add a pack using the following "<pack>" specification or using packs provided by
 
   To select a specific version use: Vendor::Pack@x.y.z
   To select the newest version of a major version use: Vendor::Pack@^x.y.z
+  To select the newest version of a major, minor version use: Vendor::Pack@~x.y.z
```

Comment:
I still have trouble with this description (same for ^) as it will be the newest / latest version only if there is no version >= major.minor.patch already installed. In the statement this check is not mentioned.